app/views/declaration/summary/additional_and_authorised_parties_section.scala.html

```diff
@@ -0,0 +1,62 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import forms.declaration.{DeclarationAdditionalActors, DeclarationHolder}
+@import models.requests.JourneyRequest
+@import models.viewmodels.HtmlTableRow
+
+@(additionalActors: Seq[DeclarationAdditionalActors], holders: Seq[DeclarationHolder])(implicit messages: Messages, journeyRequest: JourneyRequest[AnyContent])
+
+@if(additionalActors.isEmpty) {
+    @components.table_row("additionalActors", HtmlTableRow(
+        label = messages("declaration.summary.parties.additional"),
+        value = messages("site.no")
+    ))
+}
+@if(holders.isEmpty) {
+    @components.table_row("holders", HtmlTableRow(
+        label = messages("declaration.summary.parties.holders"),
+        value = messages("site.no")
+    ))
+}
+
+@if(additionalActors.nonEmpty) {
```

Comment:
Please move this next to inverted condition - would be easier to consider about this

Reply:
The logic on this page is weird. We need to display section in specific order. You're not ably to do it with if else, because sometimes holders are at the top of the page, sometimes below additional actors